app/assets/stylesheets/global.scss

```diff
@@ -106,14 +109,39 @@ ul {
 }
 
 label {
-  color: #ffffff;
+  color: $white;
   margin-left: 1em;
 }
 
 .review-page {
-  background: rgba(0, 0, 0, 0.2);
+  background: $opacity-rgba-var;
 }
 
 li {
-  color: #ffffff;
+  color: $white;
+}
+
+.new-registration {
+  margin-left: 5px;
+  width: 15em;
+}
+
+.edit-registration {
+  margin-left: 5px;
+  width: 15em;
+}
+
+.edit-password {
+  margin-left: 5px;
+  width: 15em;
+}
+
+.new-password {
+  margin-left: 5px;
+  width: 15em;
+}
+
+.log-in {
+  margin-left: 5px;
+  width: 15em;
```

Comment:
Since you're already using SASS, you _could_ dry these styles up using inheritance.